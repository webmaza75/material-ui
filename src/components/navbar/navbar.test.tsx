import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import Navbar from './navbar';

describe(`Navbar`, () => {
  it(`Navbar renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const result = renderer
      .render(<Navbar />);

    expect(result).toMatchSnapshot();
  });
});
