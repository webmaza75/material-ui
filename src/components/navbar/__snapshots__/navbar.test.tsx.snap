// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Navbar renders correctly 1`] = `
<WithStyles(ForwardRef(AppBar))
  className="makeStyles-navBar-5"
>
  <WithStyles(ForwardRef(Toolbar))
    className="makeStyles-toolbar-2"
  >
    <WithStyles(ForwardRef(IconButton))
      className="makeStyles-menuButton-3"
      color="inherit"
      edge="start"
    >
      <Logo />
    </WithStyles(ForwardRef(IconButton))>
    <WithStyles(ForwardRef(Typography))
      className="makeStyles-title-4"
      variant="h6"
    >
      Payload Monitoring
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Button))
      color="inherit"
    >
      Login
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(IconButton))
      color="inherit"
      edge="end"
    >
      <Memo(MoreVertIcon) />
    </WithStyles(ForwardRef(IconButton))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
